src/package_io/sanity_check.star

```diff
@@ -41,8 +41,8 @@ SUBCATEGORY_PARAMS = {
         "holocene_time_offset",
         "interop_time_offset",
         "fund_dev_accounts",
-        "batcher_image",
     ],
+    "batcher_params": ["image", "extra_params"],
```

Comment:
pls adjust it to be more like OP_CONTRACT_DEPLOYER_PARAMS. And most likely you gonna need to add the batcher_params to the ROOT_PARAMS.